lxd/container_lxc.go

```diff
@@ -491,19 +491,23 @@ func parseRawIdmap(value string) ([]shared.IdmapEntry, error) {
 		switch entries[0] {
 		case "both":
 			entry.Isuid = true
-			ret.AddSafe(entry)
-			ret.AddSafe(shared.IdmapEntry{
-				Isgid:    true,
-				Hostid:   entry.Hostid,
-				Nsid:     entry.Nsid,
-				Maprange: entry.Maprange,
-			})
+			entry.Isgid = true
```

Comment:
@tych0 because an idmap entry can have both Isgid and Isuid set to true, which renders as a uid and a gid entry being generated. That's in line with what the IdmapEntry parser supports for "b".